### DISCUSSION TITLE
Examples do not compile in Arduino IDE

### DISCUSSION THREAD

**Discussion by CameronCode22** [arduino-libraries/Arduino_Threads#105] [General]
Hi, I am using the rp2040 connect Nano Arduino. With the mbed os compiler. My project is using voice recognition and I2S on top of mbed_os. I now have got to stage where I need to multi thread to handle both I2S and PDM. I was trying a few examples. The .inot files are not recognised.

Am I supposed to be using a different compiler?

Thanks,

**Comment by per1234** [comment]
Hi @CameronCode22. This library can only be used with a non-standard modified version of Arduino CLI or Arduino IDE. You can download it from the links here:

https://github.com/arduino-libraries/Arduino_Threads/tree/main/docs#user-content-tooling
